packages/react-core/src/components/Menu/examples/Menu.md

```diff
@@ -692,3 +700,535 @@ class MenuOptionMultiSelect extends React.Component {
   }
 }
 ```
+
+### With drilldown
+
+```js
+import React from 'react';
+import { Menu, MenuContent, MenuList, MenuItem, Divider, DrilldownMenu } from '@patternfly/react-core';
+import StorageDomainIcon from '@patternfly/react-icons/dist/js/icons/storage-domain-icon';
+import CodeBranchIcon from '@patternfly/react-icons/dist/js/icons/code-branch-icon';
+import LayerGroupIcon from '@patternfly/react-icons/dist/js/icons/layer-group-icon';
+import CubeIcon from '@patternfly/react-icons/dist/js/icons/cube-icon';
+import {
+  MenuItemStartRollout,
+  MenuItemPauseRollout,
+  MenuItemStorage,
+  MenuItemAppGrouping,
+  MenuItemLabels
+} from './Submenus.js';
+
+class MenuWithDrilldown extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      activeItem: null,
```

Comment:
It doesn't look like `activeItem` is used in render or logic?

Reply:
`activeItem` indicates the selected item (set by onSelect)

Reply:
Right, but I only see it used 3 places in the 2 drilldown examples:
```js
this.state = {
      **activeItem: null,
      menuDrilledIn: [],
      drilldownPath: [],
      menuHeights: {},
      activeMenu: 'rootMenu'
    };
 ...
 this.onSelect = (event, itemId) => {
      if (itemId.startsWith('group:')) {
        console.log(`selected sub-menu: ${itemId.split('group:')[1]}`);
      } else {
        console.log(`selected: ${itemId}`);
        this.setState({
          **activeItem: itemId
        });
      }
    };
 ...
 const { **activeItem, menuDrilledIn, drilldownPath, activeMenu, menuHeights } = this.state;
```
Look like it's not used anywhere in the render logic and can be safely removed.

Reply:
Yeah I can remove them